README.md

```diff
@@ -1,5 +1,7 @@
 # Sindri Provers for Scroll SDK
 
+[![Build](https://img.shields.io/github/actions/workflow/status/Sindri-Labs/sindri-scroll-sdk/ci.yaml)](https://github.com/Sindri-Labs/sindri-scroll-sdk/actions)
```

Comment:
This badge shows the result of our `ci.yaml` GitHub action.

Reply:
Noice